test/integration/auth/svcaccttoken_test.go

```diff
@@ -400,6 +402,8 @@ func TestServiceAccountTokenCreate(t *testing.T) {
 			}
 
 			info := doTokenReview(t, cs, treq, false)
+			// we are not testing the credential-id feature, so delete this value from the returned extra info map
+			delete(info.Extra, apiserverserviceaccount.CredentialIDKey)
```

Comment:
If we do leave the `delete`s in place, I would check that this key is non-empty before deleting it from the map.

Reply:
the linter disagrees with 😅 ```ERROR: test/integration/auth/svcaccttoken_test.go:242:4: S1033: unnecessary guard around call to delete (gosimple)
ERROR: 			if _, ok := info.Extra[apiserverserviceaccount.CredentialIDKey]; ok {```